tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/QuietBackgroundProcessesView.xaml.cs

```diff
@@ -26,4 +27,12 @@ private async void UserControl_Loaded(object sender, RoutedEventArgs e)
     {
         await ViewModel.LoadViewModelContentAsync();
     }
+
+    private async void ShowAnalyticSummaryButton_Click(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
+    {
+        var analyticSummaryPopup = new AnalyticSummaryPopup(ViewModel.GetProcessPerformanceTable());
+        analyticSummaryPopup.XamlRoot = this.Content.XamlRoot;
+        analyticSummaryPopup.RequestedTheme = this.ActualTheme;
```

Comment:
Doing this will prevent the theme from updating if you were to have this open and change the system theme. 
Does this work?
```suggestion
        analyticSummaryPopup.RequestedTheme = this.RequestedTheme;
```

That way "Default" could be passed through instead of only Dark or Light.

Reply:
If i change to RequestedTheme, then I get a mix of Dark & Light UI.  ActualTheme seems to keep it consistent.

Reply:
Probably indicates other issues. Keep it for now but consider trying to fix it in the future.

Reply:
sticking with ActualTheme after side chat